src/stories/button.css

```diff
@@ -0,0 +1,30 @@
+.storybook-button {
```

Comment:
Just curious why these css files were added. I'm on the latest version of storybook for one of my projects and don't have these.

Reply:
Instead of upgrading storybook, I just deleted the existing one and installed a brand new copy which apparently adds this garbage file for their default example. I had so many changes I didn't catch it. Will remove.

Reply:
Huh okay. 
Maybe we should also do this for the update:
```
WARN To use the new preset, install `@storybook/preset-create-react-app` and add it to the list of `addons` in your `.storybook/main.js` config file.
```